### ISSUE TITLE
Format error for enumerations with more than 5 atomic species

### ISSUE THREAD

**Issue by wsmorgan** [msg-byu/enumlib#16]
When trying to enumerate systems with more than 5 atomic species the code produces the following error upon trying to write the output.

```
At line 1227 of file derivative_structure_generator.f90 (unit = 14, file = 'struct_enum.out')
Fortran runtime error: Unexpected end of format string
abels: ",i1,"/",i1,"/",i1,"/",i1,"/",i1,
```

**Comment by glwhart** [comment]
This just needs a more general solution. Give it a try.

On Thu, Apr 21, 2016 at 2:45 PM, Wiley Morgan notifications@github.com
wrote:

> When trying to enumerate systems with more than 5 atomic species the code
> produces the following error upon trying to write the output.
> 
> At line 1227 of file derivative_structure_generator.f90 (unit = 14, file = 'struct_enum.out')
> Fortran runtime error: Unexpected end of format string
> abels: ",i1,"/",i1,"/",i1,"/",i1,"/",i1,
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/msg-byu/enumlib/issues/16

## 

Gus Hart
http://msg.byu.edu

**Comment by wsmorgan** [comment]
Error was caused by a hardcoded size of 80 for the output format string. Any enumeration with more than 5 elements exceeds that limit. For now I set the format string variable to be limited to 200 allowing for an additional 110 elements to be included in the enumerations.

Changes will be implemented in merge request #17.
